frontend/debug.log

```diff
@@ -0,0 +1 @@
+[1102/142832.167:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
I think it might be a good idea to remove this file from your commit.

Reply:
Done